converter/internal/prometheusconvert/testdata/azure.river

```diff
@@ -0,0 +1,53 @@
+prometheus.remote_write "default" {
+	external_labels = {}
+
+	endpoint {
+		name = "remote1"
+		url  = "http://remote-write-url1"
+
+		queue_config {
+			capacity             = 2500
+			max_shards           = 200
+			max_samples_per_send = 500
+		}
+
+		metadata_config {
+			max_samples_per_send = 500
+		}
+	}
+}
+
+discovery.azure "prometheus1" {
+	subscription_id = "subscription"
+
+	managed_identity {
+		client_id = "client"
+	}
+	follow_redirects = true
+	enable_http2     = true
+}
+
+prometheus.scrape "prometheus1" {
+	targets = concat([{
+		__address__ = "localhost:9090",
+	}],
+		discovery.azure.prometheus1.targets)
```

Comment:
This looks weird, is this being formatted properly? 

I think maybe if we're concating multiple things, each set of targets should be on their own runs of lines, so that it's this instead:

```river 
targets = concat(
  [{__address__ = "localhost:9090"}], 
  discovery.azure.prometheus1.targets,
)
```

Reply:
Though if there's more than one label for the static object it should probably be

```river 
targets = concat(
  [{
     __address__ = "localhost:9090",
     foo         = "bar",
  }], 
  discovery.azure.prometheus1.targets,
)
```

Reply:
I fixed up the multiline concat formatting. The single line address example would require changes to how slice expressions are tokenized in general unless I don't reuse the generic method:

```golang
expr.SetValue([]map[string]string{keyValMap})
toks = append(toks, expr.Tokens()...)
```

Reply:
That's fine, the single line form isn't necessary. The output looks much, much better now. Thanks!